language-plutus-core/src/Language/PlutusCore/Name.hs

```diff
@@ -37,7 +35,7 @@ data Name a = Name { nameAttribute :: a
 -- those used for terms.
 newtype TyName a = TyName { unTyName :: Name a }
     deriving (Show, Lift)
-    deriving newtype (Eq, Ord, Functor, NFData, PrettyCfg)
+    deriving newtype (Eq, Ord, Functor, NFData)
```

Comment:
I assume the deriving machinery gives us a `Pretty` instance of some sort?

Reply:
Yes, but the underlying instance depends on a constraint that is in another file.